modules/auto-delete/events/voiceStateUpdate.js

```diff
@@ -0,0 +1,18 @@
+module.exports.run = async function (client, oldState) {
+    if (!client.botReadyAt) return;
+
+    const voiceChannels = client.configurations['auto-delete']['voice-channels'];
+
+    const channelConfigEntry = {...voiceChannels.find((vc) => oldState.channelId === vc.channelID)};
+    if (!channelConfigEntry.channelID) return;
+
+    const channel = await client.channels.fetch(channelConfigEntry.channelID);
+    if (!channel) {
+        return client.logger.error(`[auto-delete] ${localize('auto-delete', 'could-not-fetch-channel', {c: channel.channelID})}`);
+    }
+    if (channel.members.size) return;
```

Comment:
This relies on javascript type matching (`0` -> `null` -> `boolean`). 
This can cause issues on certain versions and reduces code-readability. I highly recommend using `dcVoiceChannel.members.size === 0` as condition

Reply:
Fixed it